### ISSUE TITLE
cartesio_planning installation

### ISSUE THREAD

**Issue by EnricoMingo** [ADVRHumanoids/soap_bar_rrt#2]
After the debian installation you should be able to compile the follwoing:

[cartesio_planning](https://github.com/ADVRHumanoids/cartesio_planning)

at ```devel``` branch.

**Comment by GaloisPaolo** [comment]
Hi guys, the link here is giving an error. Probably it s due to permission issues. Any help?

**Comment by EnricoMingo** [comment]
Hmmm... maybe, since I am not admin I can not give you access so:

[cartesio_planning-master.zip](https://github.com/ADVRHumanoids/soap_bar_rrt/files/4830915/cartesio_planning-master.zip)

**Comment by EnricoMingo** [comment]
even if maybe you need devel: 
[cartesio_planning-devel.zip](https://github.com/ADVRHumanoids/soap_bar_rrt/files/4830918/cartesio_planning-devel.zip)

**Comment by GaloisPaolo** [comment]
Installation completed!

I wanted to try some examples, but it seems that I don't have urdf files (for example if I try "roslaunch cartesio_planning example_comanplus_manipulation.launch", it says "Resource not found: cogimon_urdf").

What should I do?

**Comment by EnricoMingo** [comment]
Yes, you have to download the cogimon model [HERE](https://github.com/ADVRHumanoids/iit-cogimon-ros-pkg)

**Comment by GaloisPaolo** [comment]
Ok done, thank you!

Now launching example_comanplus_manipulation.launch I have a (red) comanplus in rviz, so I think the issue with the urdf/srdf is solved. The problem is that from the terminal it seems like the planner is trying to compute a plan without succeeding.
Am I doing sth wrong?
